### ISSUE TITLE
[feature request] close window by middle click on title bar

### ISSUE THREAD

**Issue by djshaji** [mate-desktop/marco#425]
Using this feature, a user could **close a window by middle clicking the title bar**. This already works in the Mate Panel, and in many applications for tabs. I would really appreciate it :1st_place_medal:

**Comment by lukefromdc** [comment]
Middle-click actions are configurable in org>mate>marco>general>action-double-click-titlebar in dconf-editor, though "close" is not one of the options listed. The default value is "toggle-maximize"

**Comment by sc0w** [comment]
@lukefromdc do you mean `action-middle-click-titlebar` ?

That feature can be added in the gsettings key:
`/ org / mate / marco / general / action-middle-click-titlebar`

**Comment by lukefromdc** [comment]
Yes, I meant action-middle-click-titlebar, sorry about the screwup with that

**Comment by djshaji** [comment]
Could _close_ be added to 
`/ org / mate / marco / general / action-middle-click-titlebar` ?

It wouldn't break anything and be a harmless and useful feature to include. Many, many thanks would be in order :1st_place_medal:
